pinecone/control/pinecone_interface.py

```diff
@@ -0,0 +1,553 @@
+from abc import ABC, abstractmethod
+
+from typing import Optional, Dict, Union, Literal
+
+
+from pinecone.config import Config
+
+from pinecone.core.openapi.db_control.api.manage_indexes_api import ManageIndexesApi
+
+
+from pinecone.models import ServerlessSpec, PodSpec, IndexList, CollectionList
+
+
+class PineconeDBControlInterface(ABC):
```

Comment:
I made this interface to be a home for docstrings because I was tired of looking at them while trying to edit the Pinecone class. When docs are thorough, you can't see the method signature and the method body at the same time and it annoys me every time.